h/static/scripts/directive/thread.coffee

```diff
@@ -174,8 +178,8 @@ isHiddenThread = (elem) ->
 # Directive that instantiates {@link thread.ThreadController ThreadController}.
 ###
 module.exports = [
-  '$parse', '$window', 'pulse', 'render',
-  ($parse,   $window,   pulse,   render) ->
+  '$parse', '$window', '$location', '$anchorScroll', 'pulse', 'render',
+  ($parse,   $window, $location, $anchorScroll, pulse,   render) ->
```

Comment:
Indentation: please add some spacing so that `$location` ends up directly under `'$location'`, etc.